ola build error: OSCNode.cpp:153:28: error: cannot convert 'lo_arg*' to 'lo_blob' {aka 'lo_blob_*'} in initialization

I get 

```
plugins/osc/OSCNode.cpp: In function 'int ola::plugin::osc::OSCDataHandler(const char*, const char*, lo_arg**, int, void*, void*)':
plugins/osc/OSCNode.cpp:153:28: error: cannot convert 'lo_arg*' to 'lo_blob' {aka 'lo_blob_*'} in initialization
  153 |       lo_blob blob = argv[0];
      |                      ~~~~~~^
      |                            |
      |                            lo_arg*
plugins/osc/OSCNode.cpp: In member function 'bool ola::plugin::osc::OSCNode::Init()':
plugins/osc/OSCNode.cpp:266:50: error: invalid conversion from 'int (*)(const char*, const char*, lo_arg**, int, void*, void*)' to 'lo_method_handler' {aka 'int (*)(const char*, const char*, lo_arg**, int, lo_message_*, void*)'} [-fpermissive]
  266 |   lo_server_add_method(m_osc_server, NULL, NULL, OSCDataHandler, this);
      |                                                  ^~~~~~~~~~~~~~
      |                                                  |
      |                                                  int (*)(const char*, const char*, lo_arg**, int, void*, void*)
In file included from /usr/include/lo/lo.h:32,
                 from ./plugins/osc/OSCNode.h:24:
/usr/include/lo/lo_lowlevel.h:876:72: note:   initializing argument 4 of 'lo_method_* lo_server_add_method(lo_server, const char*, const char*, lo_method_handler, const void*)'
  876 |                                const char *typespec, lo_method_handler h,
      |                                                      ~~~~~~~~~~~~~~~~~~^
make[2]: *** [Makefile:12250: plugins/osc/libolaoscnode_la-OSCNode.lo] Error 1
make[2]: Leaving directory '/home/tobiasb/.cache/rua/build/ola/src/ola-0.10.9'
make[1]: *** [Makefile:16480: all-recursive] Error 1
make[1]: Leaving directory '/home/tobiasb/.cache/rua/build/ola/src/ola-0.10.9'
make: *** [Makefile:7553: all] Error 2
==> ERROR: A failure occurred in build().
    Aborting...

```

during the build of ola.
~~I don't find anything upstream about this at https://github.com/MylesBorins/node-osc/issues?q=is%3Aissue+is%3Aopen+cannot+convert.~~
~~That's why I wanted to ask here first.~~ Does anyone know what this is about?

--- a/packages/ola/.gitignore
+++ b/packages/ola/.gitignore
@@ -1,1 +1,2 @@
 /fix-protobuf-v20.patch
+/fix-liblo.patch

--- a/packages/ola/PKGBUILD
+++ b/packages/ola/PKGBUILD
@@ -13,15 +13,19 @@ depends=()
 makedepends=(avahi cppunit doxygen flake8 libftdi-compat liblo libmicrohttpd ncurses protobuf-21
   python-numpy python-protobuf util-linux-libs)
 source=("$pkgbase-$pkgver.tar.gz::https://github.com/OpenLightingProject/$pkgbase/archive/refs/tags/$pkgver.tar.gz"
-        "fix-protobuf-v20.patch::https://github.com/OpenLightingProject/$pkgbase/pull/1875.patch")
+        "fix-protobuf-v20.patch::https://github.com/OpenLightingProject/$pkgbase/pull/1875.patch"
+        "fix-liblo.patch::https://github.com/OpenLightingProject/$pkgbase/pull/1954.patch")
 sha256sums=('d2a80d907f5a0a5169583060bd32aa0fdbb2fdfe18d1194dd922595ab6d447b5'
-            '6e57bb39c94981f62da6cd6b5c2744ace8ebe4b7b9fa8ee383800c1e3b8d6742')
+            '6e57bb39c94981f62da6cd6b5c2744ace8ebe4b7b9fa8ee383800c1e3b8d6742'
+            'b72b5e337546d93fe3873ab1aae7427e59e121ae0cb50bf5eeeb372ef6c5e411')
 options=(!lto)
 
 prepare() {
   cd $pkgbase-$pkgver
   # https://github.com/OpenLightingProject/ola/pull/1875
   patch -p1 -i ../fix-protobuf-v20.patch
+  # https://github.com/OpenLightingProject/ola/pull/1954
+  patch -Np1 -i ../fix-liblo.patch
 }
 
 build() {

--- a/packages/ola/PKGBUILD
+++ b/packages/ola/PKGBUILD
@@ -4,7 +4,7 @@
 pkgbase=ola
 pkgname=(ola ola-docs)
 pkgver=0.10.9
-pkgrel=4
+pkgrel=5
 pkgdesc='Open Lighting Architecture for controlling entertainment lighting equipment'
 arch=(x86_64 aarch64)
 url='https://www.openlighting.org'

